src/components/about/about.css

```diff
@@ -0,0 +1,55 @@
+.about {
+  background-image: url(./svg/ellipse.svg);
+  background-position: 120px -120px; /*TODO: find rem equivalent*/;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.title {
+  padding: 0 1rem;
+  font-size: larger;
+}
+
+.top-text{
+  padding: 1rem 2rem;
+}
+
+
+.bottom-section {
+  background-color: #D81921;
+}
+
+.person {
```

Comment:
Can we rename this padding to `padding-left-1` or in some fashion that could be then be reusable?